examples/applications/plot_species_distribution_modeling.py

```diff
@@ -31,7 +31,7 @@
 ----------
 
  * `"Maximum entropy modeling of species geographic distributions"
-   <http://www.cs.princeton.edu/~schapire/papers/ecolmod.pdf>`_
```

Comment:
[http://www.cs.princeton.edu/~schapire/papers/ecolmod.pdf](http://www.cs.princeton.edu/~schapire/papers/ecolmod.pdf) is still accessible to me